src/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .analyse_md import analyse_md
-
 __all__ = [
```

Comment:
shouldn't there be import statements here? I can't do 
```python
import GEMDAT
GEMDAT.plot_displacement_per_site(...)
```
right now

Reply:
Yep, good point!